tests/test_subset.py

```diff
@@ -133,6 +164,9 @@ def test_time_incomplete_years(self):
         np.testing.assert_array_equal(out.time.max().dt.day, 30)
 
 
+@pytest.mark.skipif(
```

Comment:
Why not use `pytest.importorskip` instead ?

Reply:
For some reason, `pytest.importorskip` was not registering the tests as skipped in the log. Using this method clearly shows in the pytest evaluation the specific tests that were skipped. I want to know which files and tests are being skipped and this seemed like the better approach to get that done.

Reply:
Good point. See https://github.com/pytest-dev/pytest/issues/5551#issuecomment-508453960 for a work around.